Doc/library/uuid.rst

```diff
@@ -301,3 +342,19 @@ Here are some examples of typical usage of the :mod:`uuid` module::
    >>> uuid.UUID(bytes=x.bytes)
    UUID('00010203-0405-0607-0809-0a0b0c0d0e0f')
 
+
+.. _uuid-cli-example:
+
+Command-Line Example
+--------------------
+
+Here are some examples of typical usage of the :mod:`uuid` command line interface:
+
+    # generate a random uuid - by default uuid4() is used
+    >>> python -m uuid
```

Comment:
We generally use `>>>` in python repls, I suggest replacing this with `$` not indicate that this is a raw shell.

Reply:
I was getting some build errors after this - `Could not lex literal_block as "python3". Highlighting skipped.`. I tried passing it a directive right after the `::` but looked like it didn't pick it up, but using a separate code-block with a directive seems to work now.